src/mono/mono/mini/mini-ops.h

```diff
@@ -1184,6 +1184,15 @@ MINI_OP3(OP_MULX_HL64, "mulxhl64", LREG, LREG, LREG, LREG)
 
 #endif
 
+#if defined(TARGET_X86) || defined(TARGET_AMD64)
+MINI_OP3(OP_X86_LDIVREM, "long_divrem", LREG, LREG, LREG, LREG)
```

Comment:
These could use some comments as to what they exactly do and how the 2 opcodes work together.